api/envoy/extensions/transport_sockets/tls/v3/tls.proto

```diff
@@ -124,8 +125,25 @@ message DownstreamTlsContext {
   OcspStaplePolicy ocsp_staple_policy = 8 [(validate.rules).enum = {defined_only: true}];
 }
 
+// TLS key log configuration.
+// The key log file format is "format used by NSS for its SSLKEYLOGFILE debugging output" (text taken from openssl man page)
+message TlsKeyLog {
+  // The path to save the TLS key log.
+  string tls_keylog_path = 1 [(validate.rules).string = {min_len: 1}];
```

Comment:
The field names in this message probably don't need `tls_keylog_` prefixes, since it's already clear from the message that they're inside of that they are settings for TLS key logging.  How about calling them `logfile_path`, `local_address_range`, and `remote_address_range`?

Reply:
will fix.